frontend/src/views/pages/MainPage.tsx

```diff
@@ -1,49 +1,87 @@
-// src/App.tsx
-
 import React, { useEffect, useState } from "react";
 import "./MainPage.css";
 import Footer from "../components/Footer/Footer";
 import HeroSection from "../components/HeroSection/HeroSection";
 import ProcessStep from "../components/ProcessStep/ProcessStep";
 import Table from "../components/Table/Table"
-import logo from '@/assets/Square_Cash_app_logo.png'; 
-const apiUrl = import.meta.env.VITE_API_URL;
+import Title from "../components/Title/Title";
+import GraphImage from "../assets/GraphImage.png"
+import DownArrow from "../assets/DownArrow.png"
+import Header from "../components/Header/Header";
 
 const MainPage: React.FC = () => {
 
   return (
-    <div className="MainPage">
-      <div className="App">
-      <div className="sticky-logo">
-        <img src={logo} alt="Logo" />
-      </div>
-      {/* Hero Section */}
-      <HeroSection
-        heading="Target fraud detection bias at the source."
-        body="Some sort of subtitle text here. Check out our handy guide ↓ if you need any help!"
-      />
 
-      {/* Process Step / Instruction Section */}
-      <ProcessStep
-        title="Upload your dataset."
-        stepnum={1}
-        body="The format of the dataset must be a parquet file with the following mandatory columns highlighted in the table below."
-      />
-      <Table />
-      <ProcessStep
-        title="View our feedback."
-        stepnum={2}
-        body="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean molestie mi id sapien posuer."
-      />
-      <ProcessStep
-        title="Adjust and try again!"
-        stepnum={3}
-        body="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean molestie mi id sapien posuer."
-      />
+    <div className="main-page">
+      <Header />
+
+      <Title label="BYE BYE BIAS"/>
 
+      <div className="arrow-container">
+       <a href="#about">
+          <img src={DownArrow} alt="Down Arrow" id="down-arrow" />
+        </a>
+      </div>
+
+      <section id="about">
+        <div className="flex-container">
+          <HeroSection
+            heading={
+              <>
+              Target <span className="highlight">fraud detection</span> bias at the source.
+              </>
+            }
+            body={
+              <>
+              Visualize and alleviate bias in transaction datasets fed 
+              into fraud detection AI models. Check out our <a href="#handy-guide"><span className="highlight"> handy guide ↓</span>
```

Comment:
I'm not too sure but accessibility in regards to this might have to be considered further, I remember Sam talking about how it is important to label buttons and links (and differentiate the two), not sure if this is sufficient for a screen reader.

Also, not sure if this is a diff ticket but he mentioned it must also be accessible if you don't have a mouse, so by the arrow keys.